src/serenity/config.hpp

```diff
@@ -170,7 +164,7 @@ class SerenityConfig {
    * Getter for section.
    * In case of no section - create such.
    */
-  std::shared_ptr<SerenityConfig> getSection(std::string sectionKey) {
+  std::shared_ptr<SerenityConfig> getSection(const std::string& sectionKey) {
```

Comment:
How do you store nested configs if Variant value does not support `SerenityConfig` type?